test/WebJobs.Script.Tests/Rpc/LanguageWorkerChannelTests.cs

```diff
@@ -108,6 +109,18 @@ public void SendWorkerInitRequest_PublishesOutboundEvent()
             Assert.True(traces.Any(m => string.Equals(m.FormattedMessage, _expectedLogMsg)));
         }
 
+        [Theory]
+        [InlineData(RpcLog.Types.Level.Information, RpcLog.Types.Level.Information)]
+        [InlineData(RpcLog.Types.Level.Error, RpcLog.Types.Level.Error)]
+        [InlineData(RpcLog.Types.Level.Warning, RpcLog.Types.Level.Warning)]
+        [InlineData(RpcLog.Types.Level.Trace, RpcLog.Types.Level.Information)]
+        public void SendSystemLogMessage_PublishesSystemLogMessage(RpcLog.Types.Level levelToTest, RpcLog.Types.Level expectedLogLevel)
```

Comment:
what about invalid LogLevel? RpcLogLevel might not convert to a valid LogLevel. What happens then?

Reply:
Trace is an invalid LogLevel. Expected is Information. (That's the last InlineData)